pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -404,14 +413,41 @@ def create(self, request, path):
 
         data = {
             "task": reverse(
-                "collection-imports-detail",
-                kwargs={"path": path, "pk": async_result.pk},
+                settings.ANSIBLE_URL_NAMESPACE + "collection-imports-detail",
```

Comment:
Injecting the `URL_NAMESPACE` here, makes me think, we need to do the same when we connect the views to paths in `urls.py`.

Reply:
I mean, the last argument is used to look for the corresponding urlpattern in urls.py. Yes.
And i was thinking, what might happen, if that setting stops being `""`. I bet, the reverse lookup would fail to find the url_pattern if they are not prefixed in the same way. (Matter of experiment...)